### ISSUE TITLE
flask mara_pipelines.ui.run_interactively does not work

### ISSUE THREAD

**Issue by hz-lschick** [mara/mara-example-project-1#25]
In the [Readme.md](https://github.com/mara/mara-example-project-1/blob/master/README.md#running-the-etl) it is written that I can use `flask mara_pipelines.ui.run_interactively`.

This does not work:

![image](https://user-images.githubusercontent.com/46843047/90774078-314ab300-e2f7-11ea-9926-24e535c79826.png)

**Comment by martin-loetzsch** [comment]
Hi @hz-lschick. 

Is this still a problem?

**Comment by hz-lschick** [comment]
@martin-loetzsch Yes. tested it again with the current version and checked the code but could not find the source of the issue

**Comment by jankatins** [comment]
@hz-lschick Can you just run `flask` Sometimes it spits out a better error message...

**Comment by hz-lschick** [comment]
Hmm, don't know what you mean.

I can run 'flask mara_pilelines.ui.run --help' without any issue.
How do I just call 'flask'?

**Comment by martin-loetzsch** [comment]
Ah. A while ago Flask changed how the names of these cli commands get derived from function names. So 

```flask mara_pipelines.ui.run_interactively```

became

```mara_pipelines.ui.run-interactively```

You can see all available cli commands by just running `flask`

**Comment by jankatins** [comment]
@hz-lschick just `flask` (or `flask --help` -> that should usually show you any import problems, e.g. a missing import for the package which holds the command = no command shows. Or no command at all = missing spec for the app/app.py file). 

But I just saw that there is a minus-vs-underscore issue in your screenshot:  it should be `mara_pipelines.ui.run-interactively` (minus in the last part, underscore in the earlier parts). Could it be that?

**Comment by hz-lschick** [comment]
Yes, with `flask mara_pipelines.ui.run-interactively` it works just fine. The Readme should be fixed [here](https://github.com/mara/mara-example-project-1/blame/master/README.md#L88)

`flask` works as well as it should:
![image](https://user-images.githubusercontent.com/46843047/97455275-f8067300-1937-11eb-982a-6064f5d2fa67.png)
